jekyll/_cci2/server/operator/troubleshooting-and-support.adoc

```diff
@@ -15,6 +15,34 @@ This document describes an initial set of troubleshooting steps to take if you a
 
 toc::[]
 
+[#generate-support-bundle]
+== Generate Support Bundle
+A support bundle is used by CircleCI engineers to diagnose and fix any issues you are experiencing. They are typically requested when you open a ticket.
+
+To generate a support bundle, You need to follow below steps - 
+
+=== Prerequisites
+. First, make sure circleci-server is deployed and you have access to the cluster/namespace through kubectl.
```

Comment:
@atulsingh0 can we add some commands here that people can copy to make this really straightforward for people?

Reply:
sure